docs/conf.py

```diff
@@ -0,0 +1,70 @@
+"""Configuration file for the Sphinx documentation builder."""
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+# -- Project information -----------------------------------------------------
+from typing import Any, Dict
+
+project = "MiniWoB++"
+copyright = "2023"
+author = "Farama Foundation"
+
+# The full version, including alpha/beta/rc tags
+release = "0.0.1"
```

Comment:
This needs to be dynamically set depending on the miniwob version
https://github.com/Farama-Foundation/Gymnasium/blob/f6d41e85f9965fbd2258a8b34855ae434bb70b75/docs/conf.py#L32